sql/moz-fx-data-shared-prod/fivetran_costs_derived/monthly_connector_costs_v1/query.sql

```diff
@@ -0,0 +1,74 @@
+WITH connector_mar AS (
+  SELECT
+    destination_id,
+    measured_month,
+    connector_name,
+    SUM(paid_active_rows) AS paid_active_rows,
+    SUM(free_active_rows) AS free_active_rows,
+    SUM(total_active_rows) AS total_active_rows
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.incremental_mar_v1`
+  GROUP BY
+    destination_id,
+    connector_name,
+    measured_month
+),
+total_mar AS (
+  SELECT
+    destination_id,
+    measured_month,
+    SUM(paid_active_rows) AS paid_active_rows,
+    SUM(total_active_rows) AS total_active_rows
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.incremental_mar_v1`
+  GROUP BY
+    destination_id,
+    measured_month
+),
+monthly_connector_costs AS (
+  SELECT
+    destinations.destination_name,
+    measured_month,
+    connector_mar.connector_name,
+    connector_mar.paid_active_rows,
+    connector_mar.free_active_rows,
+    connector_mar.total_active_rows,
+    ROUND(
+      connector_mar.paid_active_rows / NULLIF(total_mar.paid_active_rows, 0) * 100,
+      2
+    ) AS percentage_of_total_paid_mar,
```

Comment:
Interesting point ... I added both versions but am hesitant towards the total, since at the moment not all destinations are added and they might not be .. that's why per destination feels better to me.